src/linear_algebra/finsupp.lean

```diff
@@ -290,28 +292,30 @@ end
 
 /-- Lift a family of linear maps `M →ₗ[R] N` indexed by `x : α` to a linear map from `α →₀ M` to
 `N` using `finsupp.sum`. This is an upgraded version of `finsupp.lift_add_hom`.
-We define this as an additive equivalence. For a commutative `R`, this equivalence can be
-upgraded further to a linear equivalence. -/
-def lsum : (α → M →ₗ[R] N) ≃+ ((α →₀ M) →ₗ[R] N) :=
+-/
+def lsum : (α → M →ₗ[R] N) ≃ₗ[S] ((α →₀ M) →ₗ[R] N) :=
```

Comment:
This pattern is going to pop up a few more times (I just implemented it for `is_basis.constr`). Should we make a library note that explains what `S` is doing here?

Reply:
A library note is probably a good idea. Where do you think it belongs?

Reply:
It doesn't matter too much for me. Perhaps change the implementation notes in the module doc for `linear_algebra/prod.lean` into a library note, since it is a relatively low-level file? Another location could be `smul_comm_class`.

Reply:
I vote for `smul_comm_class`. Otherwise LGTM.
bors d+

Reply:
Library note added